### DISCUSSION TITLE
Add (or show me) a possibility to give an instant feedback on copilot suggestions.

### DISCUSSION THREAD

**Discussion by Wh1teCl0ud** [community/community#35518] [Copilot Conversations]
Yeah, what the copilot can do now, is already very cool.
But could you give the ability to leave two types of feedback on its suggestions:
1. To give a comment on a suggestion to people working on the copilot, for example, to not give a specific code as a suggestion to anybody else. Cuz I saw online some examples when the copilot gives secret keys as a suggestion. Not a cool thing 😉 
2. (In case if it doesn't count what I wrote on my own instead of its suggestions) to correct the code suggestion itself, if I know or found anything better than the copilot suggests

**Comment by iamcjwilliams** [comment]
@Wh1teCl0ud Thank you for your feedback!
